### ISSUE TITLE
Version 2.1/Webextention : Firefox doesn't save the modified URL when configured to delete cookies on close

### ISSUE THREAD

**Issue by canoine** [Athou/commafeed-firefox#7]
Hello,

I updated the Commafeed addon for Firefox to 2.1, as it's available on the official Firefox repository.
Since this update, the addon doesn't save my own Commafeed URL. It always displays the default URL (http://www.commafeed.com), and it keeps trying to reach http://www.commafeed.com, so I can't use it.

The strangest thing is that the Firefox "extensions.firefox@commafeed.com.url" parameter is set to the right value, i.e. my Commafeed server URL.

I use Firefox 56 (64 bits) on Gentoo Linux.

Is there a new parameter to set ?

**Comment by Athou** [comment]
I don't know, it seems to work fine for me in Firefox 57.
Did you click on the Save button?

**Comment by canoine** [comment]
Yes, I did. It says "Options saved." during one second, but it saves nothing.

Edit : that works using a new profile. So there certainly is a problem with a modified parameter on my profile.
Disabling all the addons, but Commafeed, doesn't help.
I think I have to compare all the modified parameters one by one, now... I'll try this when I'll have some time to do that.

I meet the same behiaviour on my wife's computer, using Firefox 57 64 bits on ubuntu 16.04 LTS.

**Comment by canoine** [comment]
OK, I got it, but I don't understand the logic behind this behaviour.

When I configure Firefox to clean the cookies on exit (In french : "Les conserver jusqu'à" : "la fermeture de Firefox". In english, that may be "Keep until" : "Firefox close", or something like that), it does not save the modifications made during the current session. But it keeps the value set before, even if it isn't the default one.

So, to save my own URL, I need to set "keep cookies until expiration", modify the Commafeed URL parameter, close and reopen Firefox, and then set "keep cookies until Firefox close" again.

The seems more a Firefox bug than the addon one... That said, I modify the issue name.
